src/main/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpStep.java

```diff
@@ -44,53 +47,96 @@ public boolean execute(String jobName, Date jobRunDate) throws InterruptedExcept
 
     protected boolean processCurrentFileAndExtractPdpFeedFromSAEFile(File currentFile) {
         boolean success = true;
-        if (!currentFile.isDirectory()) {
+        if (!currentFile.isDirectory() && isTextFile(currentFile)) {
```

Comment:
Is this isTextFile check is still needed since you have specified the txt fileExtension in the standardAccountExtractFileType definition? I think it would make sense to use this in conjunction with the BatchInputFileServiceImpl.listInputFileNamesWithDoneFile method to get the files to be processed.